### DISCUSSION TITLE
How to add a fallback for a lazy loaded component (dynamic import)?

### DISCUSSION THREAD

**Discussion by preetmishra** [nuxt/nuxt#26624] [Questions]
With slow networks, a lazy loaded component could take a while to load. Without a fallback, there's nothing on the screen while the component loads. I went through the [documentation](https://nuxt.com/docs/guide/directory-structure/components#dynamic-imports) but couldn't find anything regarding fallbacks.

**Comment by zakarialounes** [comment]
Up
